app/javascript/components/App.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
+import Greeting from './greetings'
+
+const App = () => {
+  return (
+    <Router>
+      <Switch>
+        <Route exact path="/" component={Greeting} />
+      </Switch>
```

Comment:
- [ ] Great work :+1: making your API endpoint return random messages as expected but please, your greetings component is not rending on this route since I am not seeing the random messages on the root route `http://localhost:3000/` which you have set here for the greetings component render. :blush: